src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/http-foundation": "~2.7|~3.0",
         "symfony/http-kernel": "~2.7|~3.0",
         "symfony/filesystem": "~2.7|~3.0",
-        "symfony/routing": "~2.7|~3.0",
+        "symfony/routing": "~3.0",
```

Comment:
Looks like this is not the root of the pb: components=low are still failing but subtree-splitting won't fix them. Maybe you need to add a higher req for one of the 2nd level deps? See <2.7 vers:
https://travis-ci.org/symfony/symfony/jobs/47218155#L3011